### PR TITLE
fix CI issues in ibm forked bmcweb

### PR DESCRIPTION
**Pull request by geissonator**
Signed-off-by: Andrew Geissler <geissonator@yahoo.com>

### CODE REVIEW

**Comment by geissonator** [comment]
jenkins run tests please

**Comment by geissonator** [comment]
jenkins run tests please

**Comment by geissonator** [comment]
These are a result of our downstream patches so nothing to point to upstream for these fixes.

File: redfish-core/lib/event_service.hpp [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 #include <app.hpp>
 #include <registries/privilege_registry.hpp>
+
```
Comment (gtmills) [inline redfish-core/lib/event_service.hpp:21]: Looks like bad merging

File: redfish-core/lib/event_service.hpp [hunk 2]
```diff
@@ -32,9 +33,9 @@ static constexpr const std::array<const char*, 3> supportedRetryPolicies = {
 
 #ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
 static constexpr const std::array<const char*, 8> supportedResourceTypes = {
-     "IBMConfigFile",  "Task",    "BroadcastService",
```
Comment (gtmills) [inline redfish-core/lib/event_service.hpp:35]: https://github.com/ibm-openbmc/bmcweb/commit/d81fe7d076f49684714620962a10b13a002f4e07

File: redfish-core/lib/hypervisor_system.hpp [hunk 3]
```diff
@@ -162,7 +162,8 @@ inline std::string getIfAttributeName(const std::string& ifaceId)
     {
         return "if0";
     }
-    else if (ifaceId == "eth1")
+
+    if (ifaceId == "eth1")
```
Comment (gtmills) [inline redfish-core/lib/hypervisor_system.hpp:166]: https://github.com/ibm-openbmc/bmcweb/commit/36b09cbdc1e6e82efce26f92cc8d3039304eabfb

**Review by gtmills — APPROVED** [review]

**Comment by geissonator** [comment]
jenkins run tests please

**Pull request merged**